src/converter.ts

```diff
@@ -37,7 +45,7 @@ export interface IConverter<R, V> {
   defaultControlled: Controlled<R, V>;
   neverRequired: boolean;
   preprocessRaw(raw: R, options: StateConverterOptionsWithContext): R;
-  isEmpty(raw: R): boolean;
+  isEmpty(raw: R, options: StateConverterOptionsWithContext): boolean;
```

Comment:
Just to be sure; adding `options` here are "not optional", is a breaking change?

Reply:
Correct. When writing your own converter, it has to follow the definition specified in the type ConverterOptions. `options` is an optional argument there.